Fixes MongoDB atomicapp

cdrage opened the pull request:
https://hub.docker.com/r/centos/mongodb-26-centos7/ caused us to switch from 
MONGODB_USERNAME to MONGODB_USER.

Atomic App does not interpret backslashes in the Docker artifact, those were
removed.

cdrage committed:
Changes from username or user in CentOS MongoDB Docker image.

Due to changes to: https://hub.docker.com/r/centos/mongodb-26-centos7/

MONGODB_USERNAME is now MONGODB_USER

This fixes issue
https://github.com/projectatomic/nulecule-library/issues/64

surajssd commented:
@cdrage don't you think that Atomic App should parse backslashes? I mean this is a basic thing to have?

dustymabe commented:
@surajssd perhaps. but I don't think it does currently do that. @surajssd can you confirm that it currently doesn't work with atomicapp and then open an issue against atomicapp? We can decide what to do with it from there.

dustymabe commented:
Can we update the readme as well to refer to projectatomic vs kolab?

tuxdna commented:
Also pull changes from this PR: https://github.com/projectatomic/nulecule-library/pull/67

I have made some additional changes.

cdrage commented:
I'll close this in favour of #67

surajssd commented:
@dustymabe yep I will test with backslashes!

cdrage committed:
Remove backslashes from Docker artifact.

Atomic App does not interpret the backslashes correctly in the Docker
command and thus is unable to run the Docker artifact / provider.

surajssd commented:
@cdrage @dustymabe [Atomic App issue #617](https://github.com/projectatomic/atomicapp/issues/617) reported an issue regarding back slashes.

cdrage commented:
I updated the README.

This also changes the current maintainer to `+MAINTAINER Red Hat, Inc. <container-tools@redhat.com>`

I've included a commented out section for the original source of @kanarip 's image :+1:

dustymabe commented:
@cdrage - LGTM

@kanarip do you want to give this PR a look before we merge?
